arch/arm/src/kinetis/kinetis_edma.c

```diff
@@ -1162,14 +1178,11 @@ int kinetis_dmach_start(DMACH_HANDLE handle, edma_callback_t callback,
 void kinetis_dmach_stop(DMACH_HANDLE handle)
 {
   struct kinetis_dmach_s *dmach = (struct kinetis_dmach_s *)handle;
-  irqstate_t flags;
```

Comment:
why remove the lock

Reply:
I have implemented it in kinetis_dmaterminate.